src/test/cypress/cypress/integration/Archives.spec.js

```diff
@@ -0,0 +1,49 @@
+
+describe ('Archives screen tests',function () {
+
+    before('Set up configuration', function () {
+        cy.loadTestConf();
+        cy.deleteTestCards();
+        cy.deleteAllArchivedCards();
+        cy.sendTestCards();
+    });
+
+    after('Clean', function () {
+        cy.deleteTestCards();
+        cy.deleteAllArchivedCards();
+    });
```

Comment:
Remove this clean as it is clean on test startup

Reply:
Done.